convert2rhel/pkghandler.py

```diff
@@ -365,6 +369,21 @@ def preserve_only_rhel_kernel():
     return
 
 
+def install_gpg_keys():
+    loggerinst = logging.getLogger(__name__)
+    loggerinst.info("Importing Red Hat GPG keys ...")
+    gpg_path = os.path.realpath(os.path.join(os.path.dirname(__file__),
+                                "data/version-independent/gpg-keys/*"))
+    for gpg_key in glob.glob(gpg_path):
+        shutil.copy(gpg_key, _rpm_gpg_dir)
+        output, ret_code = utils.run_subprocess(
+            'rpm --import %s' % os.path.join(_rpm_gpg_dir, gpg_key),
+            print_output=False)
+        if ret_code != 0:
+            loggerinst.critical("Unable to copy GPG key: %s" % output)
+            return
```

Comment:
This method `install_gpg_keys()` returns None in both cases if ret_code != 0 and if ret_code == 0.
Is it correct behavior?

Reply:
Actually, calling the critical logger causes exit of the tool: https://github.com/oamg/convert2rhel/blob/master/convert2rhel/logger.py#L100
It's quite unexpected, I know. We can change that. But for now, the `return` here can be removed.